foundation-faq.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: usage
+title: Quarkus at CommonHaus FAQ
+subtitle: Frequently Asked Questions
+permalink: /foundation/faq
+---
+
+## Why move to a foundation?
+
+Enable us to welcome more contributions from a diverse range of developers and organisations.
+
+## Which Quarkus projects are moving to a foundation?
+
+Initially, all projects under the https://github.com/quarkusio organisation.
+Future consideration would apply to parts or all of Quarkiverse and Quarkus Tools projects. 
+
+## What is Commonhaus?
+
+See https://www.commonhaus.org/.
+
+## Why Commonhaus?
+
+The main reason for choosing Commonhaus over another foundation was its governance framework, which emphasises project self-governance.
+See, in particular, its guiding principles.
+
+## Why not [put your favourite foundation here]?
+
+See Commonhaus’ own [FAQ](https://www.commonhaus.org/about/faq.html).
+
+## What does the move consist of exactly?
+
+The move consists of donating intellectual property owned by Red Hat and related to Quarkus projects.
+The donated intellectual property includes in particular:
+
+ * the Quarkus trademark;
+ * related domain names.
+
+It does not include copyright, since Red Hat does not have exclusive claims to copyright on Quarkus code.
+
+## Is Red Hat reducing funding for Quarkus development?
+
+No.
+
+## How is Red Hat’s involvement in Quarkus changing?
+
+Red Hat’s involvement in Quarkus remains as strong as ever.
+
+The Quarkus team is expecting increased collaboration between Red Hat and other companies, which may require setting time aside to get new contributors on board.
+
+## What is changing in practice for contributors to Quarkus projects?
+
+Commonhaus requires Quarkus to adhere to its [policies](https://www.commonhaus.org/policies/), including in particular its [intellectual property policy]https://www.commonhaus.org/policies/ip-policy/) and [trademark policy](https://www.commonhaus.org/policies/trademark-policy/).
+
+This is great news for you all, as it legally binds the project to remain open-source forever.
+
+Commonhaus also requires having a code of conduct, and provides a recommended one, but each project has flexibility on such details so we could decide to customise it.
+
+Quarkus contributors can opt to become Commonhaus Members; this is not a requirement to participate in the Quarkus project but is a requirement to help define Commonhaus.
```

Comment:
```suggestion
Quarkus contributors can opt to become [Commonhaus Members](https://www.commonhaus.org/community/#become-a-member); membership is not a requirement to participate in the Quarkus project, but members can also help shape the future of Commonhaus itself.
```

Not totally sure what this sentence is saying. Is it saying that if people want to help shape the future direction of Commonhaus they need to become members? I'll assume so.

Reply:
yeah - missed this one before pushing but I'll adjust it in follow up commit